front-page.php

```diff
@@ -10,26 +12,66 @@
   <section id="posts">
 
 <?php
-if( have_posts() ) {
-  while( have_posts() ) {
-    the_post();
+  $args = array(
+    'post_type' => 'programacion',
+    'posts_per_page' => 1,
+    'meta_query' => array(
+      'relation' => 'AND',
+      array(
+        'key'     => '_igv_start_time',
```

Comment:
shouldnt this be end time otherwise after the start of the latest it will not show?